doc/source/tutorial/csgraph.rst

```diff
@@ -47,9 +47,9 @@ later::
     >>> word_list = [word for word in word_list if len(word) == 3]
     >>> word_list = [word for word in word_list if word[0].islower()]
     >>> word_list = [word for word in word_list if word.isalpha()]
-    >>> word_list = map(str.lower, word_list)
+    >>> word_list = list(map(str.lower, word_list))
```

Comment:
There are unicode characters in the dictionary (probably accents), so would need to decode to UTF-8.
I thought about using that approach, but when the words are printed out they are changed to bytes, i.e. `and` becomes `b'and'`.